docs/volume/edit-volume.md

```diff
@@ -0,0 +1,24 @@
+---
+sidebar_position: 2
+sidebar_label: Edit a Volume
+title: ""
+keywords:
+- Volume
+Description: Edit volume from the Volume page.
+---
+
+# Edit a Volume
+
+After creating a volume, you can edit your volume by clicking the `⋮` button and selecting the `Edit Config` option.
+
+## Expand a Volume
+
+You can expand a volume by increasing the value of the `Size` parameter directly.
```

Comment:
```suggestion
You can expand a volume by increasing the value of the `Size` parameter directly.
To prevent the expansion from interference by unexpected data R/W, Harvester supports `offline` expansion only. You must shut down the VM or detach the volume first if it is attached to a VM, and the detached volume will automatically attach to a random node with [maintenance mode](https://longhorn.io/docs/1.3.2/concepts/#22-reverting-volumes-in-maintenance-mode) to expand automatically.
```